presto-main/src/test/java/com/facebook/presto/type/TestJsonOperators.java

```diff
@@ -368,6 +369,25 @@ public void testCastWithJsonParse()
         assertCastWithJsonParse("{\"a\"\n:1,  \"b\":\t2}", "MAP<VARCHAR,INTEGER>", mapType(VARCHAR, INTEGER), ImmutableMap.of("a", 1, "b", 2));
         assertInvalidCastWithJsonParse("{\"[1, 1]\":[2, 2]}", "MAP<ARRAY<INTEGER>,ARRAY<INTEGER>>", "Cannot cast JSON to map(array(integer),array(integer))");
         assertInvalidCastWithJsonParse("{true: false, false:false}", "MAP<BOOLEAN,BOOLEAN>", "Cannot cast to map(boolean,boolean).\n{true: false, false:false}");
+
+        assertCastWithJsonParse(
+                "{\"a\"  \n  :1,  \"b\":  \t  [2, 3]}",
```

Comment:
Can you talk with @highker to figure out why he formatted the test input this way? If it's intentional, add comment to explain why. If not, please reformat them.

Reply:
comment added :)